packages/codegen/src/snapshot.ts

```diff
@@ -23,11 +23,12 @@ interface ServiceOpenApiOptions extends PathOption {
   filename?: string;
 }
 
+const defaultFilename = 'cognite-openapi-snapshot.json';
```

Comment:
Could we keep this hidden as this file shouldn't be edited by a user.  Adding `.json` suffix sounds good. I like the `api` => `openapi` change as well.

Reply:
Why do you think it needs to be hidden? There are files like `package-lock.json` or `yarn.lock` which are similar

Reply:
`lock` kinda deters people from manually editing it. The reason I made it hidden was to avoid people editing it. Why do you want to make it visible?

Reply:
I don't think the fact that its "hidden" will make people not change it. You do modify files like `.gitignore` right?

I think it looks odd to be "hidden" and it doesn't need to be.

I think the "snapshot" part of the name is the best we have that should give a hint that it's not a file to be manually updated.

Reply:
> I think it looks odd to be "hidden" and it doesn't need to be.

It seems to me this is a personal preference, and I don't see why this trumps a decision I made a week or two ago. It's hidden:
 - to deter users from editing it
 - it's owned by a command and is therefore placed in the "background" rather than in the front with all the code/config files

Reply:
I've re-hidden the file.